Dockerfile

```diff
@@ -1,6 +1,13 @@
 ARG GO_VERSION=1.16
 ARG BATS_VERSION=v1.3.0
 
+# XXX: Debian buster includes libseccomp < 2.5.0 and we need >= 2.5.0. For that
+# reason, we use the script "script/install-libseccomp.sh" to install a newer
+# libseccomp here. However, Debian bullseye will include libseccomp 2.5.1.
+# Therefore, whenever bullseye is released and the go team publishes an image on
+# that, we can use it and remove the script/install-libseccomp.sh script
```

Comment:
bullseye golang images are already available -- can we switch to bullseye?

Reply:
Ok, the commit was super simple and I see you just did it. I cherry-picked it here, as you asked [here][link] then :D


[link]: https://github.com/opencontainers/runc/pull/2682#discussion_r691697814